test/e2e_node/system/docker_validator.go

```diff
@@ -70,17 +70,29 @@ func (d *DockerValidator) validateDockerInfo(spec *DockerSpec, info types.Info)
 		}
 	}
 	if !matched {
+		// catch if docker is 1.13+
+		ver := `1\.(1[3-9])\..*|\d{2}\.\d+\.\d+-[a-z]{2}`
+		r := regexp.MustCompile(ver)
+		if r.MatchString(info.ServerVersion) {
+			fmt.Println("getting here")
+			d.Reporter.Report(dockerConfigPrefix+"VERSION", info.ServerVersion, good)
+			w := fmt.Errorf(
+				"docker version is greater than the maximum version validated by the node team. docker version: %s",
```

Comment:
Users running this won't know who the "node team" is.  I'd do something like:

"docker version is greater than the most recently validated version. Docker version: %s. Max validated version: %s"

Ideally we could point to a URL about what the known issues might be.  @dchen1107 @yujuhong  -- does the node team keep any documentation about the latest status of what has been validated?